### ISSUE TITLE
Is your training data set the same as your test set

### ISSUE THREAD

**Issue by Lincoln20030413** [sefibk/KernelGAN#78]
Thankss for your reply for my last question.
I also have another question. Is your training data set the same as your test set because in your code there is only one input path.
Thanks a lot.

**Comment by sefibk** [comment]
It is a unique type of training - if you want to fully understand, please read the paper.
In short, the model is trained only on the test image (single image, not a test set).
There is no training set.
It is valid since the test image has no label, the model learns the patches of the image and realizes how to upscale it.

**Comment by Lincoln20030413** [comment]
Thanks a lot. I got it.
